adagucserverEC/CRequest.cpp

```diff
@@ -2163,11 +2188,27 @@ int CRequest::process_all_layers() {
       return 1;
     }
   } else if (dataSources[j]->dLayerType == CConfigReaderLayerTypeLiveUpdate) {
-    // Render the current time in an image for testing purpose / frontend development
-    CDrawImage image;
-    layerTypeLiveUpdateRenderIntoDrawImage(&image, srvParam);
-    printf("%s%c%c\n", "Content-Type:image/png", 13, 10);
-    image.printImagePng8(true);
+    CDBDebug("in special liveupdate case with timesteps %d", dataSources[j]->getNumTimeSteps());
+    if (dataSources[j]->cfgLayer->DataPostProc.empty()) {
```

Comment:
Can you place this code inside layerTypeLiveUpdateRenderIntoDrawImage? We are trying to make CRequest.cpp a bit smaller